RecoMuon/MuonIdentification/src/DTTimingExtractor.cc

```diff
@@ -126,15 +125,9 @@ DTTimingExtractor::fillTiming(TimeMeasurementSequence &tmSequence, reco::TrackRe
   GlobalVector momv(mom.x(), mom.y(), mom.z());
   FreeTrajectoryState muonFTS(posp, momv, (TrackCharge)muonTrack->charge(), theService->magneticField().product());
 
-  // get the DT segments that were used to construct the muon
-  std::vector<const DTRecSegment4D*> range = theMatcher->matchDT(*muonTrack,iEvent);
-  
-  if (debug) 
-    std::cout << " The muon track matches " << range.size() << " segments." << std::endl;
-  
   // create a collection on TimeMeasurements for the track        
   std::vector<TimeMeasurement> tms;
-  for (std::vector<const DTRecSegment4D*>::iterator rechit = range.begin(); rechit!=range.end();++rechit) {
+  for (std::vector<const DTRecSegment4D*>::const_iterator rechit = segments.begin(); rechit!=segments.end();++rechit) {
```

Comment:
move to range based loop
(also below at L161)

Reply:
Same comment, see no reason to change that loop. Please clarify.